### ISSUE TITLE
Get your previous work on github

### ISSUE THREAD

**Issue by DoctorMikeReddy** [PlaceholderGames/2016-VnD_Game#1]
Can you each post an issue describing what you have done so far, please.

**Comment by BojanStankovic** [comment]
Unfortunately, we still haven't come to an agreement on what the game should be. At the point it is most likely that we are going to develop a game inspired by Lemmings in which a player has an option to use a VR mode for immersed god-like feeling. The game is also playable without the VR headset. Difference from the original game of Lemmings is that our world is 3D and our actors can move in different ways (still remains to be discussed in more detail). I have done some research on effects of motion sickness while moving in a VR environment. It turns out that conventional movement causes motion sickness because person's body is stationary while the brain receives information of movement.

Until now we had 3 meetings where we discussed how to solve the problem of motion sickness and what kind of game should we go with.

**Comment by ghost** [comment]
Previously I have met with the group at group meetings to discuss the games direction and theme, We've come to an agreement on a Populous / Lemmings inspired 3D God game, this game will also be playable on a VR headset.  I have been working through the Unreal Engine 4's Tutorials and studying some C++ sample codes from the Unreal Forums to get a feel of the package and to gain an understanding of how to code certain features we require for the game.  We're currently allocating tasks to each member in the team and assigning deadlines.

**Comment by DoctorMikeReddy** [comment]
Still not had anything from ScaryEme or napiorek!!!

**Comment by napiorek** [comment]
So far we had w three meetings, where the aim was to come up with a game concept. Encourage by Bojan, we decided to create a VR game. On the very first meeting, we agreed that we all will do some research, to find out what are the limitation we could face, eg. Movement in VR can be unpleasant for certain players. 
Other limitation we faced is the controller. Initially we decided to use xbox/psp controller to allow user input.
Two more meetings were needed to fully understood the limitations that come with VR and not using the VR controllers.
Currently we decided what the main features of the game should be and assigned one each for every one of us.  
The plan is to have a few simple game mechanics to be able to start developing a map/level.
